src/storageClients/CassandraTableStorageManager.js

```diff
@@ -0,0 +1,48 @@
+'use strict';
+
+const Promise = require('bluebird');
+
+const BATCH_LIMIT = 10;
+
+module.exports = {
+
+  prepareInsertTopic: function(topic) { //TODO: the insert statement to reflect the final topic schema
+    return {
+      query: 'INSERT INTO topics (id, topic, value) VALUES (?, ?, ?)',
+      params: [ topic.id, topic.topic, topic.value ]
+    };
+  },
+
+  batch: function(client, queries){
+    return new Promise((resolve, reject) => {
+      Promise.all(getBatchPromises(client, queries))
```

Comment:
I tried it, but it didn't pass my test :( So I don't think it works, unless I may have missed something.